package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "snowtify",
-  "version": "0.0.5",
+  "version": "0.0.6",
```

Comment:
Same question about whether this is should be a minor bump rather than patch.

Reply:
Changed...